packages/client/tsconfig.base.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "declaration": true,
+    "declarationMap": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "importHelpers": true,
+    "lib": ["ESNext"],
+    "module": "ESNext",
+    "moduleResolution": "Node",
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitReturns": true,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
```

Comment:
I had to turn off `noUnusedLocals` and `noUnusedParameters` otherwise ts complains about the `fragments.generated` file, as it still has empty SDK generated. any idea how to disable ts check for a single generated file?

Reply:
@cesarenaldi any ideas on how to solve the above?

Reply:
We could try to model the `codegen` config in a way that it doesn't generate sdk for `fragment.graphql`

Reply:
I tried that, but we have two plugins here that are not aware of each other:
- `typescript-graphql-request` - https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-graphql-request
- `near-operation-file-preset` - https://the-guild.dev/graphql/codegen/plugins/presets/near-operation-file-preset

I didn't find anything in the docs to skip SDK generation for a selected file.
A solution could be to add some dummy query to `fragments.graphql` to generate SDK we don't use but that satisfies the `ts` checks.

Reply:
in fact dummy query worked fine